testRadius/main.go

```diff
@@ -17,7 +17,7 @@ func main() {
 	fmt.Printf("test start ... \n")
 	packet := radius.New(radius.CodeAccessRequest, []byte(`secret`))
     rfc2865.UserName_SetString(packet, "208920100001105")
-//	rfc2865.UserName_SetString(packet, "1234")
+	rfc2865.UserName_SetString(packet, "1234")
```

Comment:
This is a change that's outside the scope of the commit title. Was it intended to be committed here?